logging/test.ts

```diff
@@ -17,21 +17,21 @@ class TestHandler extends logging.handlers.BaseHandler {
   }
 }
 
-test(function testDefaultLoggingMethods() {
-  logging.debug("Foobar");
-  logging.info("Foobar");
-  logging.warning("Foobar");
-  logging.error("Foobar");
-  logging.critical("Foobar");
+test(function testDefaultlogMethods() {
+  log.debug("Foobar");
```

Comment:
Cool - shall we also rename the directory too now? Or do you want to do that in another PR?

Reply:
Let's finish stuff in this PR and rename directory in next one.